decidim-debates/app/views/decidim/debates/versions/show.html.erb

```diff
@@ -1,11 +1,20 @@
-<div class="row">
+<% content_for :aside do %>
+  <%= cell(
+    "decidim/versions_list",
+    versioned_resource,
+    version_path: proc { |version_index| url_for(action: :show, id: version_index) },
+    i18n_scope: "decidim.meetings.versions"
```

Comment:
@entantoencuanto this scope is not correct, is using meetings

Reply:
Removed scope to use generic one